k8s/helm/vl3/templates/vl3-nse-config.yaml

```diff
@@ -15,5 +15,6 @@ data:
       vl3:
        ipam:
           prefixpool: {{ .Values.ipam.prefixPool | quote }}
+          prefixlength: {{ .Values.ipam.prefixLength | quote }}
```

Comment:
change the rest of the fields ad you are good to go

Reply:
done